kong/plugins/aws-lambda/schema.lua

```diff
@@ -36,9 +40,11 @@ return {
         } },
         { aws_key = {
           type = "string",
+          encrypted = ENCRYPTED,
         } },
         { aws_secret = {
           type = "string",
+          encrypted = ENCRYPTED,
```

Comment:
I watched the demo meeting. iirc it is smart enough to detect whether existing data is encrypted or not, correct? so no need for migrations?

Reply:
Correct. New records (with encryption enabled) will be written in encrypted format; existing records will continue to work as expected in plaintext until they are written to (eg via PATCH)